test/pr_stacktrace_test.erl

```diff
@@ -22,56 +22,42 @@ bad_arg() ->
     integer_to_list(1.0).
 
 pr_stacktrace_throw_test() ->
-    Result = try
+    Got = try
         make_throw()
     catch
         ?EXCEPTION(Class, Reason, Stacktrace) ->
             lager:pr_stacktrace(?GET_STACK(Stacktrace), {Class, Reason})
     end,
-ExpectedPart = "
-    pr_stacktrace_test:pr_stacktrace_throw_test/0 line 26
-    pr_stacktrace_test:make_throw/0 line 16
-throw:{test,exception}",
-    ?assertNotEqual(0, string:str(Result, ExpectedPart)).
-
+    Want = "pr_stacktrace_test:pr_stacktrace_throw_test/0 line 26\n    pr_stacktrace_test:make_throw/0 line 16\nthrow:{test,exception}",
```

Comment:
When I clone this repo on `win32`, my git settings check out files with `\r\n` endings, which makes this test fail due to embedded `\r` characters in the long string. This is why these expected strings are on a single line now.